browser/js/global.js

```diff
@@ -1595,11 +1595,16 @@ function getInitializerClass() {
 					global.docURL.includes('data/idle/calc/') ||
 					global.docURL.includes('data/multiuser/calc/');
 
+				// FIXME: it doesn't work in calc under cypress
 				if (L.Browser.cypressTest && isCalcTest)
 					global.enableAccessibility = false;
 
+				if (L.Browser.cypressTest)
+					window.prefs.set('accessibilityState', global.enableAccessibility);
+
 				var accessibilityState = global.prefs.getBoolean('accessibilityState');
-				accessibilityState = accessibilityState || (L.Browser.cypressTest && !isCalcTest);
+				if (accessibilityState === undefined)
+					accessibilityState = global.enableAccessibility;
```

Comment:
You can specify the default value of something in the prefs handler like so
```js
global.prefs.getBoolean('accessibilityState', global.enableAccessibility);
```

The default default value is `false`, making your code equivalent to 

```js
var accessibilityState = global.prefs.getBoolean('accessibilityState');

if (false) // accessibilityState will never be `undefined`
	accessibilityState = global.enableAccessibility;
```

The reason it wasn't like that before was that accessibilityState was *always* true if it was in localStorage, although that probably wasn't the best for a cypress test anyway